pkg/apis/proto/daemon/daemon.proto

```diff
@@ -71,6 +71,6 @@ service DaemonService {
   };
 
   rpc GetVertexMetrics (GetVertexMetricsRequest) returns (GetVertexMetricsResponse) {
-    option (google.api.http).get = "/api/v1/namespace/{namespace}/pipelines/{pipeline}/vertices/{vertex}/metrics";
```

Comment:
We don't need namespace in the url, as the daemon service is namespaced, which already have the namespace in the host url.

Reply:
Thanks